src/sentry/receivers/features.py

```diff
@@ -287,6 +287,7 @@ def record_alert_rule_created(
     project,
     rule,
     rule_type,
+    duplicate_rule,
```

Comment:
could add a default here instead of adding it in the other python files

Reply:
+1